sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -642,6 +642,13 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val CBO_ENABLED =
```

Comment:
Although it seems to influence only how the plan stats are calculated, but that can lead to changes in output plans and thus execution procedure, so I think we can say it's meant for enabling the cbo framework.